src/working-here/supporting-your-development-and-wellbeing.md

```diff
@@ -184,14 +182,13 @@ week basis to ensure you are receiving everything you need, and will NEVER get
 bored of your questions!
 
 If you feel that you’re not getting as much time with your helper as you’d like,
-please speak to the People Manager.
+please speak to the People team.
 
 As a helper, you will be vital in ensuring you have the regular catch ups - if
 you are finding it hard to schedule them, please do talk to Bus Ops.
 
 For more information on being a helper, please see the
-[checklist](https://docs.google.com/document/d/1QPv7Mg7TTlt_OyLRtstAtPQBcId-ct-uTbWqYnso2jI)
-or speak to the People Manager.
+[checklist  ](https://docs.google.com/document/d/1QPv7Mg7TTlt_OyLRtstAtPQBcId-ct-uTbWqYnso2jI)or speak to the People team.
```

Comment:
@SamJCoop676 not sure what happened there, but there are now two spaces within the link and none outside. In the editor, if you switch the toggle (screenshot below) to Markdown and copy/paste the code below, it should fix it

```
[checklist](https://docs.google.com/document/d/1QPv7Mg7TTlt_OyLRtstAtPQBcId-ct-uTbWqYnso2jI) or speak to the People team.
```

<img width="243" alt="Screenshot 2023-01-06 at 15 20 35" src="https://user-images.githubusercontent.com/40244233/211042131-a8aa23ed-3914-4339-be70-14271e69db26.png">

Reply:
should be done now?